### PR TITLE
build: release using multi-semantic-release

### PR DESCRIPTION
**Pull request by rittme**
## Description of the changes

Use multi-semantic-release to release new package versions.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -531,34 +531,9 @@ jobs:
...
-              --conventional-prerelease 
-              --exact 
-              --canary 
-              --no-git-tag-version 
-              --no-push 
-              --preid next 
-              --dist-tag next 
-              --yes
-              ${FORCE_PUBLISH}
+          command: 'yarn release'
```
Comment (vrolland) [inline .circleci/config.yml:536]: Incredible how simple it becomes!

File: packages/request-client.js/CHANGELOG.md [hunk 2]
```diff
@@ -12,127 +12,81 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
```
Comment (vrolland) [inline packages/request-client.js/CHANGELOG.md:12]: Why does this file change?

**Review by vrolland — APPROVED** [review]

**Review by benjlevesque — APPROVED** [review]
sweet!

**Review by AdamDowson — APPROVED** [review]
Great!

**Review by yomarion — APPROVED** [review]

**Pull request merged**